SasquatchMac/Config/Sasquatch.xcconfig

```diff
@@ -12,7 +12,7 @@ PRODUCT_NAME = $(TARGET_NAME)
 
 SWIFT_OBJC_BRIDGING_HEADER = SasquatchMac/SasquatchMac-Bridging-Header.h
 SWIFT_OBJC_INTERFACE_HEADER_NAME = SasquatchMacObjC-Swift.h
-SWIFT_VERSION = 3.0
+SWIFT_VERSION = 4.2
```

Comment:
Why not 5? Xcode 10.2 already supports it.

Reply:
I see that 4.2 is used in many places in the project. In order to avoid the potential risks caused by excessive syntax changes, I chose to upgrade to 4.2.

Reply:
Our minimum supported Xcode version is 10.0 https://docs.microsoft.com/en-us/appcenter/sdk/getting-started/ios. So we should be able to test building on it.